README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+# HOMEWORK 1 BIG HEADING RAWR
+My name is Max Brown and I'm a native Chicagoan. I received by Bachelor's in Environmental Health from Colorado State University. Environmental Health actually has quite little to do with the *health of the environment*, and instead focuses on *human health from environmental influences* such as air pollution, water quality, and occupational safety. I am a second year Master's of Public Policy (MPP) in the **HARRIS SCHOOL OF PUBLIC POLICY** and our brand reps instructed us to use all-caps, bold letters when referring to the school. Unfortunately I have not yet found a way to also include the "DJ air horn" sound affect when writing the name of the school. I hope to work for local government or for a non-profit and use policy to help make the world a more equitable place. I am taking three classes this quarter:
+1. This class of course!
+2. Financial Investments for Public Policy
+3. Writing for Public Policy
+
+[Here, linked in, is a link to my LinkedIn](https://www.linkedin.com/in/maxwellmbrown94/)
+
+and here is a picture of me in a hole in the rock of the Little Grand Canyon of Southern Illinois in Shawnee National Forest this summer:
+![LGC Hole Picture](https://github.com/maxwellmbrown/hw01/blob/master/Little%20Grand%20Canyon%20Hole%20Picture.jpeg)
+
+## SECONDARY _*r e f l e c t i o n*_ SECTION
+
+The actual markdown part of this assignment was quite easy (or at least it seems so so far). The markdown guide provided was extremely simple and easy to implement for the various formatting tricks asked for. The GitHub interactions have also been quite easy, with the workflow video being very useful to accomplish this. The hardest part was installing Git itself, which was quite interesting: ultimately I had to install something else (Homebrew) directly from the terminal of my computer, which I wasn't sure could possibly work, and told me of files it was downloading without any indication of the downloads' progress. Even as a 90's kid, I find it somewhat odd that today's computer geniuses opt to use some back-to-basics methods for installing programs. I felt a bit like an early 2000's movie hacker that says things like 
+> bypassing the mainframe 
+
+and
+
+>I'm In.
+
+The one proper installer I found for Git didn't work for some very obscure technojargon reason that was a fright to behold. Another tricky part I just added in was adding an image: I'm not sure if the URL will work or not. Getting the readme pushed back to GitHub was another unexpected challenge, since I encountered an error message when trying to push despite everything seeming OK. It turns out the culprit was adding a photo file on GitHub while not having it as a local file, so I needed to pull the new addition onto my local folder first before pushing the whole thing back to GitHub. 
```

Comment:
That is a good reminder for me. I encountered similar issues uploading the photos, and I used the generated link instead.